docs/platforms/javascript/guides/nuxt/install/dynamic-import.mdx

```diff
@@ -1,24 +1,36 @@
 ---
-title: Dynamic Import (default)
-sidebar_order: 1
+title: Dynamic Import (experimental)
+sidebar_order: 3
 description: "Learn about how the Nuxt SDK leverages dynamic input() in the build output."
 ---
 
 ## Understanding the `import()` expression
 
 The `import()` expression (also called "dynamic import") allows conditional and flexible module loading in ESM.
 For the Sentry Nuxt SDK, it provides an approach to initialize server-side configuration before application startup.
+
+The Sentry Nuxt SDK will be initialized before any other code runs and the Nitro server runtime code will be loaded later because of `import()`ing it.
 This early initialization is required to set up the SDK's instrumentation of various libraries correctly.
 
 ## Initializing Sentry with Dynamic `import()`
 
-By default, the Sentry Nuxt SDK includes a Rollup plugin which wraps the server entry file with a dynamic `import()`.
+Enable the dynamic `import()` by setting `autoInjectServerSentry`:
+
+```typescript {filename:nuxt.config.ts} {3}
+export default defineNuxtConfig({
+  sentry: {
+    autoInjectServerSentry: 'experimental_dynamic-import'
+  },
+})
+```
+
+After setting this, the Sentry Nuxt SDK adds a Rollup plugin which wraps the server entry file with a dynamic `import()`.
```

Comment:
l: This feels like a bit too much detail. Even the concept of `server entry file` is confusing, it sounds a bit like our server config file.

Maybe just saying that the build system will take care of ensuring import order using dynamic imports?